content/content-tool/tool/src/java/org/sakaiproject/content/tool/CopyrightDelegate.java

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright (c) 2003-2018 The Apereo Foundation
+ *
+ * Licensed under the Educational Community License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *             http://opensource.org/licenses/ecl2
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sakaiproject.content.tool;
+
+import org.apache.commons.lang.StringUtils;
+import org.sakaiproject.entity.api.ResourceProperties;
+import org.sakaiproject.entity.api.ResourcePropertiesEdit;
+import org.sakaiproject.util.ParameterParser;
+
+/**
+ * Used to centralize any copyright related tasks
+ * See SAK-39953
+ *
+ * @author bbailla2
+ */
+public class CopyrightDelegate
+{
+	private String copyrightInfo;	// Copyright info takes its value from the dropdown (eg. "I hold copyright")
+	private String copyrightStatus;	// Copyright status takes its value from the text area (ie. when "Use copyright below." is selected)
+	private boolean copyrightAlert;	// copyrightAlert is... ?
+
+	public CopyrightDelegate() {}
+
+	/**
+	 * Overload of captureCopyright; puts properties into this class's members
+	 * @param params
+	 */
+	public void captureCopyright(ParameterParser params)
+	{
+		captureCopyright(params, null);
+	}
+
+	/**
+	 * Captures copyright properties from the ParameterParser
+	 * @param params the ParameterParser to pull the copyright properties from
+	 * @param li the copyright properties will be set on li if it exists, otherwise they are stored in this class' members
+	 */
+	public void captureCopyright(ParameterParser params, ListItem li)
+	{
+		String copyright = StringUtils.trimToNull(params.getString("copyright"));
+		if (copyright == null)
+		{
+			// do nothing -- there must be no copyrightDialog
```

Comment:
Could just return early here and avoid the rightward drift for the rest of the method